test/local/routes/sign.js

```diff
@@ -182,6 +185,7 @@ describe('/certificate/sign', () => {
   })
 
   it('with unverified session that does not require verification', () => {
+    mockRequest.query.service = 'foo'
```

Comment:
These were depending on `query.service` being set as a side-effect of previous tests.